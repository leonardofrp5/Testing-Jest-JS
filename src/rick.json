{
  "id": 1,
  "name": "Leonardo Romero",
  "status": "Vivo",
  "species": "Humano",
  "gender": "Male"
}