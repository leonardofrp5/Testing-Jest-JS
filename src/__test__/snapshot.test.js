import { getCharacter } from '../snapshot';
import rick from '../rick.json';

describe('test whit snapshot', () => {
  it('test request', () => {
    expect(getCharacter(rick)).toMatchSnapshot();
  });
});