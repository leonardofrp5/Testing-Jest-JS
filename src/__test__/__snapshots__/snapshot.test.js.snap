// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test whit snapshot test request 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Leonardo Romero",
  "species": "Humano",
  "status": "Vivo",
}
`;
